.vsts-ci.yml

```diff
@@ -3,19 +3,30 @@ trigger:
 - release/*
 
 variables:
-  _TeamName: Roslyn-Project-System
-  _DotNetArtifactsCategory: Toolset
-  ${{ if eq(variables['System.TeamProject'], 'public') }}:
-    PB_PublishBlobFeedKey:
-    PB_PublishBlobFeedUrl:
-    _DotNetPublishToBlobFeed: false
-    _PublishUsingPipelines: false
-    _PublishType: none
-  ${{ if ne(variables['System.TeamProject'], 'public') }}:
-    PB_PublishBlobFeedUrl: https://dotnetfeed.blob.core.windows.net/dotnet-toolset/index.json
-    _DotNetPublishToBlobFeed: true
-    _PublishUsingPipelines: true
-    _PublishType: blob
+  - name: _TeamName
+    value: Roslyn-Project-System
+  - name: _DotNetArtifactsCategory
+    value: Toolset
+  - ${{ if eq(variables['System.TeamProject'], 'public') }}:
+    - name: PB_PublishBlobFeedKey
+    - name: PB_PublishBlobFeedUrl
```

Comment:
These probably need some value. That's why these are showing up in your pipeline:
```
Pipeline configuration error:
Object reference not set to an instance of an object.
```

Reply:
This should have 
```
value: 
```